Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN set -eux; \
     \
     # tools used by Pimcore
     apt-get install -y iproute2 unzip zip; \
+    apt-get install -y ssmtp; \
```

Comment:
Before marking this pull requests as _ready for review_ we need to test it:

- [ ] Install `ssmtp` in hosted environments on php fpm container 
- [ ] Check if this has negative side effects there / ensure sending mail will still work (even w/o configuring `mailhub` in `/etc/ssmtp/ssmtp.conf`)